src/resources/templates/baseline/logstash/pipeline/filebeat.conf.j2

```diff
@@ -0,0 +1,40 @@
+# Input.
+input {
+  beats {
+    port => 5044
+    host => "0.0.0.0"
+  }
+}
+
+# Filter(s).
+filter {
+  if ("proxy-traffic-logs" in [tags]) {
+
+    json {
+      source => "message"
+    }
+
+    date {
+      match => ["ts", "UNIX"]
+      target => "@timestamp"
+    }
+
+    # The `tls` field can be either an `object` or `bool`.
+    # Elasticsearch does not support multiple types for indexes.
+    # Easier to just drop.
+    mutate {
+      remove_field => ["ts", "tls"]
+    }
+
+  }
+}
+
+# Output.
+output {
+  if ("proxy-traffic-logs" in [tags]) {
+    elasticsearch {
+      hosts => "{{ settings.reverse_proxy.services.monitoring.elasticsearch_url | default("elasticsearch:9200") }}"
```

Comment:
Doesn't `hosts`  need to be an array? I thought they usually looks like this:

```
input { stdin { } }
...
output {
  elasticsearch { hosts => ["localhost:9200"] }
}
```
https://www.elastic.co/guide/en/logstash/6.2/config-examples.html

Reply:
My guess is it supports either (in `https://bitbucket.org/brightsparklabs/teraflow/src/develop/subprojects/cli/src/resources/templates/baseline/logstash/pipeline/filebeat.conf.j2` we have it as just a string).
However I'll convert to array to match the official docs.